### ISSUE TITLE
Get RNA-seq quote (coral)

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#1926]
Corals for 116 _stranded_ RNA-seq libraries 

Sending RNA 

20M reads

**Comment by sr320** [comment]
Would hope we could get this number with Azenta https://d.pr/i/SzaMHH

**Comment by kubu4** [comment]
Can you please share link for promo? I can't seem to find that.

**Comment by sr320** [comment]
https://web.azenta.com/promotions?utm_medium=email&_hsenc=p2ANqtz-8nQwvqLYT_f2vLkTUwnFcWiFYEtmNUL-9YxEznRwEYqtbkL_0YoyBqMCSO2q1gIWwxMtF1Mtzyn5Bx7APBTIjAp5sOhA&_hsmi=310339430&utm_content=310319944&utm_source=hs_email

**Comment by kubu4** [comment]
Quote request, with promo code, submitted.

**Comment by kubu4** [comment]
:crossed_fingers: 

Just saw in the fine print that this pricing is limited to new academic customers...

**Comment by kubu4** [comment]
Azenta:

Looks like they applied a different promotion (15% off RNA-seq). Here's the quote:

`$17,433.50` (~ $150/sample)

[30-1047560508_R2.pdf](https://github.com/user-attachments/files/15842892/30-1047560508_R2.pdf)

**Comment by kubu4** [comment]
Have requested quote from UW NGCC.

**Comment by sr320** [comment]
@kubu4 anything back from UW?

**Comment by kubu4** [comment]
Nope. Sent a follow up email.

**Comment by sr320** [comment]
lets move ahead with Azenta quote - getting PO etc

**Comment by kubu4** [comment]
PO request initiated.

**Comment by kubu4** [comment]
Have the PO number.

Need the following sample submission form filled out:

[30-1047560508_NGS_Template_File.xls](https://github.com/user-attachments/files/16179753/30-1047560508_NGS_Template_File.xls)

**Comment by sr320** [comment]
PO number?

**Comment by kubu4** [comment]
PO-0100084783

**Comment by sr320** [comment]
thanks!

**Comment by sr320** [comment]
@kubu4 can you get a modified quote where they do RNA-seq AND miRNA (short RNA) seq on same RNA sample? (116 samples)

**Comment by kubu4** [comment]
sRNA-seq quote (using same set of input RNA): `$32,110`

[30-1069297013.pdf](https://github.com/user-attachments/files/16473506/30-1069297013.pdf)

To make sure this is clear, this is in _addition_ to the [previous RNA-seq quote](https://github.com/RobertsLab/resources/issues/1926#issuecomment-2192285260).

**Comment by sr320** [comment]
can you get PO for this (or will the same PO work?)

**Comment by kubu4** [comment]
PO request for sRNA-seq submitted.

**Comment by kubu4** [comment]
PO for sRNA-seq: `PO-0100091376`

**Comment by kubu4** [comment]
Sample sheet needs to be filled out:

[30-1069297013_NGS_Template_File.xls](https://github.com/user-attachments/files/16534577/30-1069297013_NGS_Template_File.xls)

**Comment by sr320** [comment]
update to 120 samples

**Comment by kubu4** [comment]
Updated sample counts submitted earlier. Will report when updates are received.

**Comment by kubu4** [comment]
Updated sRNA-seq quote:

`$33,210.00`

[30-1069297013_R1.pdf](https://github.com/user-attachments/files/16567176/30-1069297013_R1.pdf)


Sample sheet needs to be filled out:

[30-1069297013_NGS_Template_File.xls](https://github.com/user-attachments/files/16567183/30-1069297013_NGS_Template_File.xls)

**Comment by kubu4** [comment]
Updated RNA-seq quote:

`$18,028.50`

[30-1047560508_R4.pdf](https://github.com/user-attachments/files/16589378/30-1047560508_R4.pdf)

Sample sheet needs to be filled out:

[30-1047560508_NGS_Template_File.xls](https://github.com/user-attachments/files/16589383/30-1047560508_NGS_Template_File.xls)

**Comment by JillAshey** [comment]
Perfect! I'll start pulling samples today!

**Comment by JillAshey** [comment]
[e5_sequencing_samples.xlsx - azenta_rna.csv](https://github.com/user-attachments/files/16618133/e5_sequencing_samples.xlsx.-.azenta_rna.csv)
Sample sheet for Azenta RNA submission

**Comment by zdellaert** [comment]
Hi all, I was preparing our list to aliquot the RNA into plates tomorrow to send to Azenta on Monday, and realized that I've already sent 6 of these tubes to Azenta for the deep dive in April 2023 (30-789513166 and 30-852430235_R3) . So, 1) we've already sequenced them and 2) we don't have those tubes in the freezer anymore. Do we need to ask for an updated quote for fewer samples?

**Comment by zdellaert** [comment]
Unless we run into any missing samples tomorrow, this should be the final list for Azenta:

[Azenta_E5_Sequencing.xlsx](https://github.com/user-attachments/files/16631161/Azenta_E5_Sequencing.xlsx)

**Comment by AHuffmyer** [comment]
Can you direct me to the list of those that were sent previously? We need to make sure to avoid batch effects do not confound time/site/species effects. 

All samples should have another molecular sample in the freezer that has not yet been extracted. The original extractions were done on one of two molecular tubes with biopsies. I suggest we re extract these samples from the second molecular replicate to send for sequencing. Thoughts?

**Comment by zdellaert** [comment]
https://docs.google.com/spreadsheets/d/1r3oGvGsLLe9fI9mktjg0MM5yMIUpUnTeiNV45gZvLFs/edit?usp=sharing 

[30-789513166_NGS_SampleInfo_File_HP.xls](https://github.com/user-attachments/files/16631218/30-789513166_NGS_SampleInfo_File_HP.xls)

They were sent on April 5, 2023 and the whole volume was split for stranded RNA seq and small RNA seq

[37926916-62e5-4076-8fc3-68c32c61b791.pdf](https://github.com/user-attachments/files/16631226/37926916-62e5-4076-8fc3-68c32c61b791.pdf)
[402832dd-6a98-4a9f-b318-a08f1ab0e5fd.pdf](https://github.com/user-attachments/files/16631236/402832dd-6a98-4a9f-b318-a08f1ab0e5fd.pdf)

**Comment by zdellaert** [comment]
Hi all,

Just got off the phone with Ariana, who has concerns about the batch effect of having these 6 samples sequenced in a previous run, especially since they were all from the same timepoint (TP2) and site 1. The colonies are: POC-53, POC-57, POR-73, ACR-145, ACR-150, and ACR-173. 

@sr320 and Hollie, we wanted your thoughts on this batch effect issue and whether we should:

1) Re-extract RNA from these samples, possibly delaying the RNA shipment by a few days if any issues arise (if there is material left and RNA is good on extraction we could still likely get RNA shipped on Monday)

or 

2) Send the 111 samples we already have extracted and use the previous deep dive data for the 6 samples already sequenced. Ariana is worried this will cause difficulty to parse out site/timepoint differences if there is a strong sequencing batch effect given that all the deep dive samples were from one site and timepoint.

**Comment by sr320** [comment]
New extractions

**Comment by zdellaert** [comment]
Plates are done and in the freezer, ready to be shipped out on Monday. Final sample sheet for Azenta attached.
[20240816_Azenta_E5_RNA_Sequencing.xlsx](https://github.com/user-attachments/files/16641346/20240816_Azenta_E5_RNA_Sequencing.xlsx)

**Comment by AHuffmyer** [comment]
Were you able to perform new extractions using existing tissue in molecular shield tubes?

**Comment by JillAshey** [comment]
Yes that info is here: https://github.com/JillAshey/JillAshey_Putnam_Lab_Notebook/blob/master/_posts/2024-08-16-MiniprepPlus-DNA%3ARNA-extractions-E5.md

**Comment by JillAshey** [comment]
Tracking number for RNAseq package: 778050045280. Will be picked up today and delivered tomorrow morning
